pkg/orchestrator/etcd_worker.go

```diff
@@ -223,81 +222,50 @@ func (worker *EtcdWorker) syncRawState(ctx context.Context) error {
 	return nil
 }
 
-func mergePatch(patches []*DataPatch) []*DataPatch {
-	patchMap := make(map[util.EtcdKey][]*DataPatch)
-	for _, patch := range patches {
-		patchMap[patch.Key] = append(patchMap[patch.Key], patch)
-	}
-	result := make([]*DataPatch, 0, len(patchMap))
-	for key, patches := range patchMap {
-		patches := patches
-		result = append(result, &DataPatch{
-			Key: key,
-			Fun: func(old []byte) ([]byte, error) {
-				for _, patch := range patches {
-					newValue, err := patch.Fun(old)
-					if err != nil {
-						if cerrors.ErrEtcdIgnore.Equal(errors.Cause(err)) {
-							continue
-						}
-						return nil, err
-					}
-					old = newValue
-				}
-				return old, nil
-			},
-		})
+func (worker *EtcdWorker) cloneRawState() map[util.EtcdKey][]byte {
+	ret := make(map[util.EtcdKey][]byte)
+	for k, v := range worker.rawState {
+		ret[k] = v
 	}
-	return result
+	return ret
```

Comment:
What is the purpose of this clone function, the underlying byte slice is not deep cloned.